dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/TestSPIConfig.java

```diff
@@ -0,0 +1,142 @@
+/*******************************************************************************
+ * Copyright (c) 2017, 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.checkpoint.fat;
+
+import static org.junit.Assert.assertNotNull;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.SkipIfCheckpointNotSupported;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import io.openliberty.checkpoint.spi.CheckpointPhase;
+
+@RunWith(FATRunner.class)
+@SkipIfCheckpointNotSupported
+public class TestSPIConfig {
+    @Rule
+    public TestName testName = new TestName();
+
+    private static final String USER_FEATURE_PATH = "usr/extension/lib/features/";
+    private static final String USER_BUNDLE_PATH = "usr/extension/lib/";
+    private static final String USER_FEATURE_USERTEST_MF = "features/test.checkpoint.config-1.0.mf";
+    private static final String USER_FEATURE_USERTEST_JAR = "bundles/test.checkpoint.config.bundle.jar";
+
+    @Server("TestSPIConfig")
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void addTestFeature() throws Exception {
+        server.copyFileToLibertyInstallRoot(USER_FEATURE_PATH, USER_FEATURE_USERTEST_MF);
+        server.copyFileToLibertyInstallRoot(USER_BUNDLE_PATH, USER_FEATURE_USERTEST_JAR);
+
+    }
+
+    @AfterClass
+    public static void cleanup() throws Exception {
+        //server.deleteFileFromLibertyInstallRoot(USER_FEATURE_PATH + USER_FEATURE_USERTEST_MF);
```

Comment:
Did you forget to comment these back in?

Reply:
yes, should be fixed now.